modules/openxr/action_map/openxr_interaction_profile_meta_data.h

```diff
@@ -58,6 +58,8 @@
 #define XR_PATH_UNSUPPORTED_NAME "unsupported"
 
 class OpenXRInteractionProfileMetaData : public Object {
+	GDCLASS(OpenXRInteractionProfileMetaData, Object);
```

Comment:
Worth checking whether this is meant to be registered at all, as the other option would be to remove the `GDREGISTER_CLASS` call if not needed. CC @BastiaanOlij

Reply:
Good point

Reply:
Was mostly just future proofing this for when we start accessing it through GDExtensions. Not sure how I missed adding GDCLASS here so yes this is the correct fix.

Somewhat surprised at the fallout this causes, dang

Reply:
If there is no use for it at this moment, maybe it's better to hide it for now and only expose it when it becomes useful?

Reply:
I think with the work Migeran has done this class should likely be exposed properly by now.